docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -0,0 +1,95 @@
+# Training models with GPU support
+:label:`sec_tabulargpu`
+
+Training with GPU can significantly speed up base algorithms, but it is also a necessity for text and vision models where even on small tasks and the best CPUs the training can take forever without GPU acceleration. To enable GPU training, pass `num_gpus` parameter into a `fit()` call. All models
+with GPU support will automatically start using the acceleration.
+
+```{.python}
+save_path = 'agModels-predictClass'
+predictor = TabularPredictor(label=label, path=save_path).fit(
+    train_data,
+    ag_args_fit={'num_gpus': 1}
+)
+```
+
+If GPU acceleration should be enabled just on a specific models, the same parameter can be passed into model `hyperparameters`:
+
+```{.python}
+save_path = 'agModels-predictClass'  # specifies folder to store trained models
+hyperparameters = {
+    'GBM': [
+        {'ag_args_fit': {'num_gpus': 0}},  # Train with CPU
+        {'ag_args_fit': {'num_gpus': 1}}   # Train with GPU
+    ]
+}
+predictor = TabularPredictor(label=label, path=save_path).fit(
+    train_data, 
+    hyperparameters=hyperparameters, 
+)
+```
+
+## Multi-modal
+
+In :ref:`sec_tabularprediction_multimodal` tutorial we presented how to train an ensemble which can utilize tabular, text and images. If available GPU don't have enough RAM to handle default model sizes or it is needed to speedup testing, different backends can be used:
```

Comment:
GPU -> GPUs, RAM -> VRAM, different backends can be used: -> different backends can be used.

Reply:
updated